### ISSUE TITLE
createAssetAsync save the asset instead of just creating it.

### ISSUE THREAD

**Issue by moshfeu** [expo/expo#1844]
### Environment

Environment:
  OS: OS X El Capitan 10.11.6
  Node: 8.10.0
  Yarn: Not Found
  npm: 5.8.0
  Watchman: Not Found
  Xcode: Not Found
  Android Studio: 2.3 AI-162.4069837

Packages: (wanted => installed)
  expo: ^27.0.2 => 27.0.2
  react: 16.3.1 => 16.3.1
  react-native: https://github.com/expo/react-native/archive/sdk-27.0.0.tar.gz => 0.55.2

Diagnostics report:
  https://exp-xde-diagnostics.s3.amazonaws.com/moshfeu-bda84927-2d35-4674-937b-ae59344ba3ed.tar.gz

### Steps to Reproduce

1. Use a `Camera` object to take image from the app
2. Call the `MediaLibrary.createAssetAsync` with the `image.uri`output from the prev step.

### Expected Behavior

An `Asset` will be created but not stored in a default album such as **DCIM**.
**Or** I could delete this asset whenever I want. In this case the name of the default album (DCIM), will be stored in the `Constants`
**Or** there will be a method like `removeAssetFromDefaultAlbum(asset)` or something like this.

### Actual Behavior

The asset saved in my default album (**DCIM**) instead of just been created.

I'm always keep saying "default album" but maybe it's default folder.

### Some code

    const photo = await this.camera.takePictureAsync();
    const asset = await MediaLibrary.createAssetAsync(localUri);
    // in this point, the asset will added to my DCIM album

### Reproducible Demo

https://snack.expo.io/@moshfeu/createassetasync-repro

**Comment by moshfeu** [comment]
Thanks for the response @tsapeta .

> add album parameter to createAssetAsync

Sounds great 👍

I can't use your solution because if the album is not exist, I need to create an album with this asset 

    album = await MediaLibrary.createAlbumAsync(ALBUM_NAME, asset);

so, when I try to move the asset to this album, I have 2. One from the album creation and the second from the moving.

For now, I have workaround:

    const asset = await MediaLibrary.createAssetAsync(localUri);
    const DCIM_id = asset.albumId as string;
    await MediaLibrary.createAlbumAsync(ALBUM_NAME, asset);
    await MediaLibrary.removeAssetsFromAlbumAsync([asset], DCIM_id);

**Comment by moshfeu** [comment]
@tsapeta what's the status? Just closed?

**Comment by tsapeta** [comment]
@moshfeu Oh yeah, I'm sorry I closed it by accident when I merged our internal PR that (I hope) fixes this issue. It should be available to the public in the very near future.

Unfortunately, it was not possible to add album parameter to createAssetAsync as I proposed previously, but we have added `copyAsset` parameter to `createAlbumAsync` - if you pass `false` there, then you will not have to remove the asset from the original album. It actually works just like `copyAssets` in `addAssetsToAlbumAsync` function. I believe it solves your problem or at least simplifies your solution.
